e2e-test/src/test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.common.stepsdesign;
+
+import com.google.cloud.bigquery.BigQueryException;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.UUID;
+
+/**
+ * GCP test hooks.
+ */
+public class TestSetupHooks {
+  private static boolean firstFileSourceTestFlag = true;
+  private static boolean firstFileSinkTestFlag = true;
+
+  @Before(order = 1, value = "@FILE_SOURCE_TEST")
+  public static void setFileSourceAbsolutePath() {
```

Comment:
are we using file source plugin in these tests? If not, please remove irrelevant code

Reply:
Yes, we are using file source plugin.